pkg/logger/uilogger_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2022 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logger_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/spf13/cobra"
+
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/cli-runtime"
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/logger"
+)
+
+func TestNoopLogger(t *testing.T) {
+	t.Run("NoopLogger", func(t *testing.T) {
+		cmd := &cobra.Command{}
+		ctx := cli.WithCommand(context.Background(), cmd)
+		ctx = logger.StashSourceImageLogger(ctx, logger.NewNoopLogger())
+		noopLogger := logger.RetrieveSourceImageLogger(ctx)
+		if noopLogger == nil {
+			t.Errorf("Stash or Retrieve NoopLogger Failed")
+		}
+	})
+}
+
+func TestRetrieveProgressLogger(t *testing.T) {
+	t.Run("NoopLogger", func(t *testing.T) {
+		cmd := &cobra.Command{}
+		ctx := cli.WithCommand(context.Background(), cmd)
```

Comment:
you do not need cmd in context

Reply:
I have updated the test case to remove the command